### PR TITLE
itermocil 0.1.6

### PR DESCRIPTION
**Pull request by TomAnthony**
[iTermocil](https://github.com/TomAnthony/itermocil) allows you setup pre-configured layouts of windows and panes in iTerm2, having each open in a specified directory and execute specified commands. It is compatible with teamocil files designed for tmux sessions.

I did previously submit this ~7 weeks ago (https://github.com/Homebrew/homebrew/pull/42549) right after I launched it but it hadn't yet proved whether it would be popular. Since then it has gained a decent amount of popularity and continues to do so, and it would be great to get it into homebrew proper.

### CODE REVIEW

**Comment by dunn** [comment]
Looks like it's not finding pyyaml:
```
==> /usr/local/Cellar/itermocil/0.1.6/bin/itermocil -h
Traceback (most recent call last):
  File "/usr/local/Cellar/itermocil/0.1.6/libexec/bin/itermocil", line 3, in <module>
    from itermocil import main
  File "/usr/local/Cellar/itermocil/0.1.6/libexec/bin/itermocil.py", line 8, in <module>
    import yaml
ImportError: No module named yaml
Error: itermocil: failed
Failed executing: /usr/local/Cellar/itermocil/0.1.6/bin/itermocil -h
```

**Comment by TomAnthony** [comment]
I removed a bit of formula and tested but PyYaml was already linked on my machine so I missed that I needed it. I have fixed and tested again (with PyYaml missing to start with). Thanks.

**Comment by TomAnthony** [comment]
@dunn Do I need to open a new PR now this is ready for a revisit? Thanks.

File: Library/Formula/itermocil.rb [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  url "https://github.com/TomAnthony/itermocil/archive/0.1.6.tar.gz"
+  sha256 "c22dec9156cd8934713858f3efe772d9aebcba53782d53456ec6bf7040235dec"
+
+  resource "PyYAML" do
+    url "https://pypi.python.org/packages/source/P/PyYAML/PyYAML-3.11.tar.gz"
+    sha256 "c36c938a872e5ff494938b33b14aaa156cb439ec67548fcab3535bb78b0846e8"
+  end
+
+  def install
+    resource("PyYAML").stage { system "python", *Language::Python.setup_install_args(libexec/"vendor") }
```
Comment (DomT4) [inline Library/Formula/itermocil.rb]: You seem to be installing the same resource twice here, in the same place?

Comment (DomT4) [inline Library/Formula/itermocil.rb]: Ping?

**Comment by DomT4** [comment]
Closing as dead. Please feel free to resubmit as fresh with the comments above addressed and audit passing, if you'd like to. Thanks for the submission to Homebrew!

**Pull request closed without merging**